### PR TITLE
v0.3.0

### PR DESCRIPTION
**Pull request by zerious**
Sweeping changes, moving functionality to lighter-common, etc.
* Replaced "Class" with lighter-common "Type".
* Lower-cased lib files.
* Removed zeriousify dependency.

### CODE REVIEW

File: chug.js [hunk 1]
```diff
@@ -29,12 +29,12 @@ for (var property in waiter) {
...
-    return require('./package.json').version;
+    return require(__dirname + '/package.json').version;
   }
 });
 
 /**
- * Don't walk upward, and ignore DS_Store.
+ * Don't walk upward, and ignore DS_Store, etc.
  */
 api._ignorePattern = /^(\.+)(|DS_Store|gitignore)$/;
```
Comment (hasalfred) [inline chug.js]: you may want to consider ignoring bower configs later down the road

File: common/object/type.js [hunk 2]
```diff
@@ -0,0 +1,44 @@
+/**
```
Comment (hasalfred) [inline common/object/type.js]: thought i just saw this script under cedar, would this shared among lighter common as a separate node_module?

File: chug.js (hunk 1 above)
Comment (zerious) [inline chug.js]: Agreed. Ultimately, this should just read .gitignore and ignore all the right things.

**Comment by hasalfred** [comment]
LGTM

**Pull request merged**